### PR TITLE
GraphQL ID: rename `__isTypeOf` to `isTypeOf`

### PR DESCRIPTION
**Pull request by mrtnzlml**
Preparation for stable release.

### CODE REVIEW

**Comment by mrtnzlml** [comment]
Should be without breaking changes so people can migrate the code.

File: src/graphql-global-id/src/GlobalID.js [hunk 1]
```diff
@@ -27,8 +27,15 @@ export function toGlobalId(type: string, id: string | number): OpaqueIDString {
   return encode(`${type}:${id}`);
 }
 
-// TODO: find out better way how to do it (type should be just an internal detail - see evaluateGlobalIdField)
+/**
+ * TODO: remove in 1.0.0
+ * @deprecated
+ */
 export function __isTypeOf(type: string, opaqueID: mixed): boolean {
+  return isTypeOf(type, opaqueID);
```
Comment (tbergq) [inline src/graphql-global-id/src/GlobalID.js]: How about also using `warning` and log here that it is deprecated?

Comment (mrtnzlml) [reply src/graphql-global-id/src/GlobalID.js]: Do you think it's necessary? I think we have the usages somehow under control, or not? :) cc @michalsanger

Comment (tbergq) [reply src/graphql-global-id/src/GlobalID.js]: necessary, probably not, but I think it is good practise to log this in dev mode. To remind any consumers that it has been deprecated. 

Ok, we don't have that many outside of the 4 of us. So I am also fine with merging it as is.

File: src/graphql-global-id/README.md [hunk 2]
```diff
@@ -50,7 +50,11 @@ const fields = {
 };
 ```
 
-# Restoring original ID
+# `toGlobalId`
+
+TKTK
```
Comment (jaroslav-kubicek) [inline src/graphql-global-id/README.md]: what's this? 😄 

Screenshot 2020-03-31 17 44 47

Comment (mrtnzlml) [reply src/graphql-global-id/README.md]: - https://en.wikipedia.org/wiki/To_come_(publishing)
- https://www.urbandictionary.com/define.php?term=TKTK

**Pull request merged**